packages/channel-client/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     // This package is meant to emit webpack processed files
-    "noEmit": true
+    "noEmit": true,
+    // Needed for EventEmitter import
+    "allowSyntheticDefaultImports": true
```

Comment:
You can use import assignment instead.

```js
import EventEmitter = require('eventemitter3');
```

Reply:
Isn't the `require` syntax an anti-pattern for TS imports?

Reply:
I think
```
import EventEmitter = require('eventemitter3');
```
exposes that this _specific_ package has a default export for the type `EventEmitter` but not for the actual object `EventEmitter` which is non-default.

Adding `allowSyntheticDefaultImports` just masks the problem in my opinion.